### DISCUSSION TITLE
Smart Meter - Meter Number of a Itron

### DISCUSSION THREAD

**Discussion by goofy79** [arendst/Tasmota#17687] [Q&A]
Hello, 

i have a Itron Smart Meter and reat out with tasmota. The Values work well, but the Meter_Number comes with "**0a01495452030099871d**" what in a hex to ascii converter results "**ITR**" .. so the rest of the number is missing ?

I have in my script 
**1,77070100600100ff@#,Zählernummer,,Meter_Number,0**
so do i something wrong ? Or can somebody help me ? 

BR

**Comment by mh254** [comment]
No help but the same question....

**Comment by joba-1** [comment]
nothing wrong here. The serial number of my Itron has the same format and it is exactly what the device delivers: 0a01, ITR, 5 unique bytes as hex
